src/commons/TestUtils.ts

```diff
@@ -27,4 +30,24 @@ export namespace TestUtils {
   export function extractNumbersFromString(str: string): string {
     return str.replace(/[^0-9]/g, '');
   }
+
+  /**
+   * Return object of data by file path
+   *  path File located in wdio config file,
+   */
+  export function getDataFromFile(): object {
+    // @ts-ignore
+    const dataFilename = browser.config.configDataFilePath;
+    const dataFilePath = path.resolve(process.cwd(), dataFilename);
+    return require(dataFilePath);
```

Comment:
This might throw an exception in case there is no such file, maybe we should fail safely with a proper error message?

Reply:
added